### PR TITLE
payment system parts 3, 4: invitations, many updates.

### PR DESCRIPTION
**Pull request by roman-tiukh**

### CODE REVIEW

File: payment_system/migrations/0010_userproject_is_default.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def default_projects(apps, schema):
     Subscription = apps.get_model('payment_system', 'Subscription')
 
     for user in User.objects.all():
-        u2p = user.user_projects.filter(role=payment_system_models.UserProject.INITIATOR).first()
+        u2p = user.user_projects.filter(role='initiator').first()
```
Comment (IegorSoboliev) [inline payment_system/migrations/0010_userproject_is_default.py:19]: ?

File: payment_system/migrations/0010_userproject_is_default.py [hunk 2]
```diff
@@ -29,8 +29,8 @@ def default_projects(apps, schema):
             UserProject.objects.create(
                 project=new_project,
                 user=user,
-                role=payment_system_models.UserProject.INITIATOR,
-                status=payment_system_models.UserProject.ACTIVE,
+                role='initiator',
+                status='active',
```
Comment (IegorSoboliev) [inline payment_system/migrations/0010_userproject_is_default.py:33]: ?

File: payment_system/migrations/0010_userproject_is_default.py [hunk 3]
```diff
@@ -40,7 +40,7 @@ def default_projects(apps, schema):
             ProjectSubscription.objects.create(
                 project=new_project,
                 subscription=def_sub,
-                status=payment_system_models.ProjectSubscription.ACTIVE,
+                status='active',
```
Comment (IegorSoboliev) [inline payment_system/migrations/0010_userproject_is_default.py:43]: ?

File: payment_system/migrations/0010_userproject_is_default.py (hunk 1 above)
Comment (roman-tiukh) [reply payment_system/migrations/0010_userproject_is_default.py:19]: Migration should not be bound to variables, they can change their value and the database schema on the current migration has a certain state

**Review by IegorSoboliev — APPROVED** [review]

**Pull request merged**